infra/tpu-pytorch-releases/artifacts.auto.tfvars

```diff
@@ -108,6 +148,43 @@ versioned_builds = [
     package_version = "1.13"
     accelerator     = "tpu"
   },
+  {
+    git_tag         = "v2.2.0"
+    pytorch_git_rev = "v2.2.0"
+    package_version = "2.2.0",
+    accelerator     = "cuda"
+    cuda_version    = "12.0"
```

Comment:
Which CUDA version are we targeting? Should we just pick the same CUDA version `torch` will use in their default wheel?

Reply:
How about we leave 11.8 and 12.1 here?

Reply:
I think this makes sense, I found Pytorch 2.1.1 is using CUDA 12.1 by default on pypi, let's use 12.1 as well. cc @vanbasten23, to double check our stack works with CUDA 12.1

Reply:
Hi @lsy323 , do we still need CUDA 11.8 for 2.2 release?

Reply:
Hi, @will-cromar, thanks, do we know how to confirm the CUDA version `torch` will use in their default wheel? would they mention in the blog related to 2.2 release?

since there were many PRs to add specific CUDA version to 2.1 release like https://github.com/pytorch/xla/pull/5683, so there might be some requirements to build 2.2 wheel with these CUDA versions. And if build failed, we could catch the failure early too

For resources, do we want to delete triggers for 2.1 to limit the resource used for release here?